thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/common/FactorySource.java

```diff
@@ -0,0 +1,15 @@
+package com.linkedin.thirdeye.common;
+
+public enum FactorySource {
+  ALERT_FETCHER("com.linkedin.thirdeye.alert.fetcher"),
+  ANOMALY_FEED("com.linkedin.thirdeye.alert.feed"),
+  EMAIL_CONTENT_FORMATTER("com.linkedin.thirdeye.alert.content");
```

Comment:
I am not sure if hard-code the path in enum is a good idea. Considering that we might have extension jar (e.g., MP) which may put the additional classes in different packages.

We should inject this class when the ThirdEye application is initialized. In that way, we could control if we want to add extension packages.